### PR TITLE
[Platform Compatibility] Update aka.ms links

### PR DESCRIPTION
**Pull request by nimanch**
Update Platform Compatibility Tool and Documentation with correct aka.ms links

## Azure IoT Edge PR checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines and Best Practices
- [x] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).
- [x] Title of the pull request is clear and informative.
- [x] Description of the pull request includes a concise summary of the enhancement or bug fix.

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.
- Description of the pull request includes 
	- [ ] concise summary of tests added/modified
	- [ ] local testing done.  

### Draft PRs
- Open the PR in `Draft` mode if it is:
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.

_Note: We use the kodiakhq bot to merge PRs once the necessary checks and approvals are in place. When it merges a PR, kodiakhq converts the PR title to the commit title, PR description to the commit description, and squashes all the commits in the PR to a single commit. The net effect is that entire PR becomes a single commit. Please follow the best practices mentioned [here](https://chris.beams.io/posts/git-commit/#:~:text=The%20seven%20rules%20of%20a%20great%20Git%20commit,what%20and%20why%20vs.%20how%20For%20example%3A%20) for the PR title and description_

### CODE REVIEW

**Review by ggjjj — DISMISSED** [review]

File: platform-validation/docs/aziot-compatibility-get-started.md [hunk 1]
```diff
@@ -52,21 +52,25 @@ Following are the checks that are performed in the Platform Compatibility script
...
+`wget aka.ms/aziot-compatibility.sh`
+
+* To make script executable, use the following command
+ 
+`chmod +x ./aziot-compatibility.sh`
 
 * To run the script, use the following commands for help.
 
-`./aziot-compatibilty --help` or `./aziot-compatibilty -h`
+`./aziot-compatibilty.sh --help` or `./aziot-compatibilty.sh -h`
```
Comment (damonbarry) [inline platform-validation/docs/aziot-compatibility-get-started.md:63]: "compatibility" is missing the last "i" in several instances here

Comment (ggjjj) [reply platform-validation/docs/aziot-compatibility-get-started.md:63]: good catch!

Comment (nimanch) [reply platform-validation/docs/aziot-compatibility-get-started.md:63]: Thanks, I will update the aka.ms link

Comment (nimanch) [reply platform-validation/docs/aziot-compatibility-get-started.md]: Ack Comments

**Review by damonbarry — APPROVED** [review]

**Pull request merged**